net-libs/libtorrent-rasterbar/libtorrent-rasterbar-2.0.9-r1.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# https://github.com/arvidn/libtorrent/commit/785969574cfe925098f50b7e5d9ae2a9f2a9d0cf
```

Comment:
What is this saying? This is why we don't have 3.12, or...?

Reply:
> This is why we don't have 3.12

That was my intention.

Reply:
"no 3.12 b/c: ..."